src/components/grid/index.tsx

```diff
@@ -1,12 +1,14 @@
 import React, { FC } from 'react';
-import { SimpleGrid, SimpleGridProps } from '@chakra-ui/react';
+import { Grid as ChakraGrid, GridProps } from '@chakra-ui/react';
 
 type Props = Pick<
-  SimpleGridProps,
-  'minChildWidth' | 'columns' | 'spacing' | 'spacingX' | 'spacingY' | 'children'
+  GridProps,
+  'children' | 'templateAreas' | 'gap' | 'templateColumns'
```

Comment:
maybe you want to allow `gridTemplateRows` too. Can come in handy. We used it in `BaseGridLayout` already